api/hw/ps2.hpp

```diff
@@ -79,11 +79,25 @@ namespace hw
     static keystate_t get_kbd_vkey();
     static uint8_t    get_mouse_irq();
 
+    // if you want to control PS/2 yourself
```

Comment:
Exposing PS2 as a raw interface ? why not register the raw interface as an alternative driver ?

Reply:
PS2 only has mouse and keyboard, there is no alternative :P